src/content/guides/using-persistent-volumes-on-aws-with-efs/index.md

```diff
@@ -0,0 +1,126 @@
++++
+title = "Using Persistent Volumes on AWS with EFS"
+description = "Tutorial on how to use dynamically provisioned Persistent Volumes on a cluster running on Amazon Web Services with EFS"
+date = "2020-02-06"
+type = "page"
+weight = 50
+tags = ["tutorial"]
++++
+
+# Using Persistent Volumes on AWS with EFS
+
+If your cluster is running in the cloud on Amazon Web Services (AWS) the most common way to store data is using EBS volumes with the [dynamic provisioner](https://docs.giantswarm.io/guides/using-persistent-volumes-on-aws/). Sometimes EBS is not the optimal solution.
+
```

Comment:
- EBS: Please introduce an abbreviation on first use, like done with AWS.
- "Sometimes EBS is not the optimal solution." - is there a more positive way to put this, like "Compared to using EBS, EFS is a superior solution to this."

Reply:
+1

Reply:
If your cluster is running in the cloud on Amazon Web Services (AWS) the most common way to store data is using Elastic Block Store volumes with the [dynamic provisioner](https://docs.giantswarm.io/guides/using-persistent-volumes-on-aws/). EBS uses hardware mounts which means it's slower to mount and there are [limits](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/volume_limits.html) on the number of volumes you can attach to the same node.


does that sound better?

Reply:
Yes. And when you put (EBS) behind the long form, like in `data is using Elastic Block Store (EBS) volumes`, then it's even better and it is crystal clear what EBS stands for.